src/3rd_party.h

```diff
@@ -0,0 +1,8 @@
+#ifndef __THIRD_PARTY_H
+#define __THIRD_PARTY_H
+
```

Comment:
I would add the `#ifdef THIRDPARTY` here and never call those functions if it's false.

Reply:
And follow the standard in other header files to create this one. The define name is different and there's `extern "C" `missing

Reply:
> I would add the `#ifdef THIRDPARTY` here and never call those functions if it's false.

I think we are using the same pattern using compile time flag for signatures...like in c src/signature.c

Reply:
> And follow the standard in other header files to create this one. The define name is different and there's `extern "C" `missing

Makes sense will change in next push

Reply:
I mixed 2 comments in same line and I realized now that it's confusing.
 - The #ifdef style is good the way it is, like the signature as you said. But If you just ignore all file it's going to be easier to code and I'm ok with that. But if you want to leave like it is, it's fine too.
 - The style I was talking about is the header file. But I realized that there's not a very well established style. each file does it in a different way. So the only thing really missing is the extern.

Reply:
For point1 I will try to do what you said. I agree with you
For point2, will make sure headers have extern c

Reply:
Makes sense will change in next push